bebraven/platform
# Fix conflicting CSS b/n Platform and FormAssembly.

The FormAssembly CSS was conflicting with ours making the forms
look weird. This change removes our CSS/JS from Form Assembly forms
by overriding the links in application.scss. Note that `launch` and
`completed` actions are still shown inside an lti_canvas layout b/c
they are not Form Assembly forms. They are the bookends for launching
them.

Task: https://app.asana.com/0/1174274412967132/1199627462312685

### Test Plan:
- Load a waivers form before this change and notice that the fonts in
  certain paragraphs, especially the first, are bigger than the fonts
  in others. After this change, the fonts look just like they do when
  you view the form directly in Form Assembly
  ( e.g. https://braven.tfaforms.net/4810867?participantId=a2X1J000000mT6tUAE )
- Make sure a non-FormAssembly page still has the application.css and js.

### Screenshots (if appropriate):
#### Before

##### NDA
<img width="600px" src="https://user-images.githubusercontent.com/5596986/105768371-4e308580-5f2a-11eb-8222-da8c3e8ac20e.png">

##### Media Release
<img width="600px" src="https://user-images.githubusercontent.com/5596986/105768420-5ab4de00-5f2a-11eb-890a-3b54f2058a75.png">

#### After

##### NDA
<img width="600px" src="https://user-images.githubusercontent.com/5596986/105768442-643e4600-5f2a-11eb-9a50-f2980448ad35.png">

##### Media Release
<img width="600px" src="https://user-images.githubusercontent.com/5596986/105768468-6ef8db00-5f2a-11eb-9cab-9ae885db8333.png">

### Types of changes




### Checklist:
